edenia/antelope-tools
# Feat/non compliant improvements 1127

### Non-compliant BPs section improvements

### What does this PR do?

- Resolve #1127 
- Set white background color to BP cards.
- Change "Last time claimed" to "Last Claimed Time".
- Validate when a url is valid and the token price is available.

### Steps to test

1. Run the project locally.
2. Go to "Non compliant Block Producers" page "/non-compliant-bps".
3. Check the improvements.
4. Then, go to "Rewards Distribution". 
3. Check the improvements.

### Screenshots

#### Non-compliant BPs

![imagen](https://user-images.githubusercontent.com/66583677/213806240-a99bb9dd-fb66-423f-b215-9e809acd3d7a.png)
![imagen](https://user-images.githubusercontent.com/66583677/213804006-d4c5d823-a348-4a44-9990-38419cbe2ce8.png)

#### Rewards Distribution

![imagen](https://user-images.githubusercontent.com/66583677/213803709-fc5931ad-850e-4a0b-bd55-d8fd8c0b986c.png)
![imagen](https://user-images.githubusercontent.com/66583677/213803729-ec382cfe-66fe-493f-a73a-414d81c8e49a.png)
